### PR TITLE
fix: Unable to access nested list property

### PR DESCRIPTION
**Pull request by arajkumar**
This PR fixes https://github.com/python-restx/flask-restx/issues/91.

### CODE REVIEW

**Comment by arajkumar** [comment]
@j5awry @ziirish 
^^ ping

File: tests/test_fields.py [hunk 1]
```diff
@@ -1369,3 +1369,21 @@ def __getitem__(self, n):
 
         obj = Test("hi")
         assert fields.get_value("value", obj) == "hi"
+
+    def test_get_value_int_indexable_list(self):
+        assert fields.get_value('bar.0', {'bar': [42]}) == 42
+
+    def test_get_value_int_indexable_list_with_str(self):
+        assert fields.get_value('bar.abc', {'bar': [42]}) == None
```
Comment (abunuwas) [inline tests/test_fields.py:1377]:
```suggestion
        assert fields.get_value('bar.abc', {'bar': [42]}) is None
```

File: tests/test_fields.py [hunk 2]
```diff
@@ -1369,3 +1369,21 @@ def __getitem__(self, n):
...
+
+    def test_get_value_int_indexable_nested_list(self):
+        assert fields.get_value('bar.0.val', {'bar': [{'val': 42}]}) == 42
+
+    def test_get_value_int_indexable_tuple_with_str(self):
+        assert fields.get_value('bar.abc', {'bar': (42, 43)}) == None
```
Comment (abunuwas) [inline tests/test_fields.py:1383]:
```suggestion
        assert fields.get_value('bar.abc', {'bar': (42, 43)}) is None
```

**Comment by arajkumar** [comment]
@abunuwas @j5awry @ziirish
^^ ping

**Pull request merged**